docs/cluster-providers/mng-cluster-provider.md

```diff
@@ -9,7 +9,6 @@ const props: MngClusterProviderProps = {
     minSize: 1,
     maxSize: 10,
     desiredSize: 4,
-    version: "auto"
```

Comment:
This line should not be removed. Rather `version: KubernetesVersion.V1_25,` line should be removed if you want to update it. Applies to other part of the doc too.

Reply:
+1 seems more reasonable not to hardcode the version